src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -654,7 +654,8 @@ function disableRow(elem, row) {
 function feedbackRecipientUpdateVisibilityOptions(elem) {
     elem = $(elem);
     if (elem.val() === 'OWN_TEAM' || elem.val() === 'TEAMS' ||
-            elem.val() === 'INSTRUCTORS' || elem.val() === 'OWN_TEAM_MEMBERS') {
+            elem.val() === 'INSTRUCTORS' || elem.val() === 'OWN_TEAM_MEMBERS' ||
+            elem.val() === 'OWN_TEAM_MEMBERS_INCLUDING_SELF') {
```

Comment:
Start <code>||</code> on a new line. Can you change this condition to follow our convention?